api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -88,6 +88,10 @@ message RedisProxy {
     // downstream unchanged. This limit defaults to 100.
     google.protobuf.UInt32Value max_upstream_unknown_connections = 6;
 
+    // Enable per-command statistics per upstream cluster, in addition to the filter level aggregate
```

Comment:
nit: please ref link to the stats documentation.

Reply:
will do